app/api/templates/specs/fixtures.js

```diff
@@ -14,6 +14,7 @@ const swapTemplate = db.id();
 const relatedTo = db.id();
 const templateToBeInherited = db.id();
 const propertyToBeInherited = db.id();
```

Comment:
Please, can you remove the first six lines in this file given there aren't needed already?

Reply:
done